### PR TITLE
Translate error messages inside fty-nut-command

### PR DESCRIPTION
**Pull request by boricj**

### CODE REVIEW

**Comment by EldoreiJK** [comment]
@aquette you're right. TRANSLATE_ME is supposed to be used to identify strings in the code for translation database purposes as well as encapsulating the string into json translation string form. If you have any source that's outside of your control (std::exception from STL for example), use JSONIFY that just packs it to json translation string and won't create a translation database entry. JSONIFY should be safe to use even for TRANSLATE_MEd strings as it should check that the content is json and leave it as is.

File: src/fty_nut_command_server.cc [hunk 1]
```diff
@@ -207,7 +207,7 @@ treat_pending_commands(nut::Client &nutClient, mlm_client_t *client, PendingComm
                 try {
                     switch (nutClient.getTrackingResult(id)) {
                         case nut::FAILURE:
-                            command.errMsg = "Failure to execute command";
+                            command.errMsg = "Failed to execute command on power device.";
```
Comment (aquette) [inline src/fty_nut_command_server.cc]: I was expecting a TRANSLATE_ME() here.
@EldoreiJK could you please confirm on the actual approach to have for this PR? thx a lot

Comment (EldoreiJK) [reply src/fty_nut_command_server.cc]: If this message makes it to the UI, then yes, otherwise no need.

Comment (aquette) [reply src/fty_nut_command_server.cc]: thx @EldoreiJK

**Pull request merged**